### PR TITLE
Load bidders names from file system in lowercase

### PR DESCRIPTION
**Pull request by VeronikaSolovei9**
Bugfix: modified code to load bidder names from the file system in lowercase

### CODE REVIEW

File: config/bidderinfo.go [hunk 1]
```diff
@@ -197,7 +197,8 @@ func LoadBidderInfo(path string) (BidderInfos, error) {
 
 		bidderName := strings.Split(fileName, ".")
 		if len(bidderName) == 2 && bidderName[1] == "yaml" {
-			infos[(bidderName[0])] = info
+			// store all bidder names loaded from file system in lowercase
```
Comment (SyntaxNode) [inline config/bidderinfo.go:200]: Nitpick: Please expand on the comment to include this is mention the conversion to lower case is to match the same behavior of config.BidderInfos .

**Review by SyntaxNode — COMMENTED** [review]
Looks great. Thank you for quickly working on this @VeronikaSolovei9. Left one minor nitpick over a comment.

**Review by guscarreon — COMMENTED** [review]
Looks pretty good.

**Review by guscarreon — APPROVED** [review]
LGTM

**Review by SyntaxNode — APPROVED** [review]

**Review by guscarreon — APPROVED** [review]
LGTM

**Pull request merged**